### ISSUE TITLE
can not

### ISSUE THREAD

**Issue by lee2seek** [aamini/introtodeeplearning_labs#8]

**Comment by lee2seek** [comment]
i followed the WindowsDocker.md steps, and everythind is good. But when i open http://192.168.99.100:8888/tree and into introtodeeplearning_labs . There is nothing in introtodeeplearning_labs . Why this happened? who can help me

**Comment by aamini** [comment]
Can you share the docker run command you are executing so I can try to reproduce this error?

**Comment by lee2seek** [comment]
` docker run \
    -p 8888:8888 \
    -p 6006:6006 \
    -v /introtodeeplearning_labs:/notebooks/introtodeeplearning_labs \
    mit6s191/iap2018:labs`
this is my docker run command. my code is in the C:/Experiments/E.introtodeeplearning_labs3. First i think maybe i should modify the directory, so i enter these docker run command like :
` docker run     -p 8888:8888     -p 6006:6006     -v c:/Experiments/E.introtod
eeplearning_labs3:/notebooks/introtodeeplearning_labs     mit6s191/iap2018:labs`
But it produce an error,the error message is :
`C:\Program Files\Docker Toolbox\docker.exe: Error response from daemon: invalid mode: /notebooks/introtodeeplearning_labs.
See 'C:\Program Files\Docker Toolbox\docker.exe run --help'.`

**Comment by aamini** [comment]
You need to provide the full path to the lab folder so the first command you provided is not sufficient. The second command is closer but depends on the terminal that you are using.

Which command line are you using? Cygwin? Windows CMD? Docker Toolbox Terminal? I suggest using the Docker Toolbox terminal on windows.

**Comment by lee2seek** [comment]
i use the Docker Toolbox terminal on windows. But there is still nothing in introtodeeplearning_labs in Jupyter.

**Comment by aamini** [comment]
If you are using docker toolbox then you will need to change the absolute host path in your `docker run` command from `c:/Experiments/E.introtod eeplearning_labs3` to `/c/Experiments/E.introtod eeplearning_labs3`

**Comment by lee2seek** [comment]
Yes, i use the absolute path `/c/Experiments/E.introtod eeplearning_labs3`. the lab1 and lab2 still not in the introtodeeplearning_labs3. I think a solution to solve this problem, i upload the file and fold to introtodeeplearning_labs3. But this solution is not good. And when i close docker and re-run docker there is still nothing in the introtodeeplearning_labs3.

**Comment by aamini** [comment]
Is there supposed to be a space in between `E.introtod` and `eeplearning_labs3`?

**Comment by lee2seek** [comment]
no spacec in between E.introtod and eeplearning_labs3. i don't know why
